File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/DanhMucChucDanh.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucChucDanh } from 'src/app/shared/DanhMucChucDanh.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucChucDanhService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucChucDanh";
    }
}

File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/DanhMucChucDanhService.cs
namespace Service.Implement
{
    public class DanhMucChucDanhService : BaseService<DanhMucChucDanh, IDanhMucChucDanhRepository>
    , IDanhMucChucDanhService
    {
    private readonly IDanhMucChucDanhRepository _DanhMucChucDanhRepository;
    public DanhMucChucDanhService(IDanhMucChucDanhRepository DanhMucChucDanhRepository) : base(DanhMucChucDanhRepository)
    {
    _DanhMucChucDanhRepository = DanhMucChucDanhRepository;
    }
    }
    }

